shimmer/docs/wasp/docs/guide/evm/magic.md

```diff
@@ -22,25 +22,29 @@ functionality through the _ISC Magic Contract_.
 The Magic contract is an EVM contract deployed by default on every ISC chain, in the EVM genesis block, at
 address `0x1074000000000000000000000000000000000000`.
 The implementation of the Magic contract is baked-in in
-the [`evm`](../core_concepts/core_contracts/evm.md) [core contract](../core_concepts/core_contracts/overview.md));
+the [`evm`](../core_concepts/core_contracts/evm.md) [core contract](../core_concepts/core_contracts/overview.md);
 i.e. it is not a pure-Solidity contract.
 
 The Magic contract has several methods, which are categorized into specialized
 interfaces: `ISCSandbox`, `ISCAccounts`, `ISCUtil` and so on.
 You can access these interfaces from any Solidity contract by importing
-the [ISC library](https://github.com/iotaledger/wasp/blob/develop/packages/vm/core/evm/iscmagic/ISC.sol).
+the [`@iota/iscmagic` library](https://www.npmjs.com/package/@iota/iscmagic).
 
 The Magic contract also provides proxy ERC20 contracts to manipulate ISC base
 tokens and native tokens on L2.
 
+## Installing
+
+See the [`@iota/iscmagic` npm package](https://www.npmjs.com/package/@iota/iscmagic) for instructions.
+
 ## Examples
 
 ### Calling getEntropy()
 
 ```solidity
```

Comment:
Just FYI:

If you have examples in your repo (in the future), you can also import those with something like:

<pre>
```solidity reference
https://github.com/iotaledger/wasp/blob/develop/packages/vm/core/evm/iscmagic/ERC20BaseTokens.sol#L4-L13
```
</pre>

This would look like this for example:
<https://wiki.iota.org/shimmer/wallet.rs/how_tos/accounts_and_addresses/create_a_wallet_account/#code-example>